process/build.py

```diff
@@ -1461,12 +1461,14 @@ def radialb(self, output: bool):
                 "(tf_in_cs)",
                 build_variables.tf_in_cs,
             )
-            po.ovarin(
-                self.mfile,
-                "TF coil radial placement switch",
-                "(tf_in_cs)",
```

Comment:
I presume `tf_in_cs` is put into the MFile somewhere else and this was just a duplicate

Reply:
Yep, it was double printed